### PR TITLE
[mini] SingleParticle Docs

### PR DESCRIPTION
**Pull request by LDAmorim**
Hi all,
This mini PR adds a brief description of the `singleparticle` injection style option, to the documentation file **Docs/source/running_cpp/parameters.rst**.
Please feel free to adapt this description to how you would prefer to have it written.
Cheers,
Diana

### CODE REVIEW

File: Docs/source/running_cpp/parameters.rst [hunk 1]
```diff
@@ -270,6 +270,12 @@ Particle initialization
     * ``NRandomPerCell``: injection with a fixed number of randomly-distributed particles per cell.
       This requires the additional parameter ``<species_name>.num_particles_per_cell``.
 
+    * ``singleparticle``: Inject a single macroparticle.
+      This requires the additional parameters:
+      ``<species_name>.single_particle_pos`` (`3 doubles`, particle 3D position)
+      ``<species_name>.single_particle_vel`` (`3 doubles`, particle 3D normalized momentum, i.e. :math:`\gamma \beta`)
+      ``<species_name>.single_particle_weight`` ( `double`, macroparticle weight, i.e. nr of physical particles it represents)
```
Comment (Yin-YinjianZhao) [inline Docs/source/running_cpp/parameters.rst:277]:
```suggestion
      ``<species_name>.single_particle_weight`` ( `double`, macroparticle weight, i.e. number of physical particles it represents)
```

Maybe write out `number`?

**Review by Yin-YinjianZhao — APPROVED** [review]

File: Docs/source/running_cpp/parameters.rst [hunk 2]
```diff
@@ -270,6 +270,12 @@ Particle initialization
     * ``NRandomPerCell``: injection with a fixed number of randomly-distributed particles per cell.
       This requires the additional parameter ``<species_name>.num_particles_per_cell``.
 
+    * ``singleparticle``: Inject a single macroparticle.
+      This requires the additional parameters:
+      ``<species_name>.single_particle_pos`` (`3 doubles`, particle 3D position)
```
Comment (MaxThevenet) [inline Docs/source/running_cpp/parameters.rst:275]: Thanks for giving the dimensions/definitions of parameters below! Could you also add this for the position? e.g.,
```suggestion
      ``<species_name>.single_particle_pos`` (`3 doubles`, particle 3D position [meters])
```

File: Docs/source/running_cpp/parameters.rst [hunk 3]
```diff
@@ -270,6 +270,12 @@ Particle initialization
     * ``NRandomPerCell``: injection with a fixed number of randomly-distributed particles per cell.
       This requires the additional parameter ``<species_name>.num_particles_per_cell``.
 
+    * ``singleparticle``: Inject a single macroparticle.
```
Comment (MaxThevenet) [inline Docs/source/running_cpp/parameters.rst:273]: Just for consistency
```suggestion
    * ``SingleParticle``: Inject a single macroparticle.
```
Note that the code converts it all to lower-case, but let's be consistent with what's written above.

**Comment by ax3l** [comment]
ping @LDAmorim :)

File: Docs/source/running_cpp/parameters.rst (hunk 2 above)
Comment (LDAmorim) [reply Docs/source/running_cpp/parameters.rst]: This was a great catch @MaxThevenet ! Would you like me to add units to the definition of the `gaussian_beam` injection style (lines below) too?

**Comment by LDAmorim** [comment]
Thanks for the ping @ax3l ! I just fixed the details pointed out by @MaxThevenet .

**Review by MaxThevenet — APPROVED** [review]
👍

**Pull request merged**